### PR TITLE
CLN: E242 space after tab, enforce all passing rules

### PR DESCRIPTION
**Pull request by bashtage**
Tiny white space only changes for E242
Enfore all other flake8 rules that are already passing

### CODE REVIEW

**Comment by bashtage** [comment]
@jbrockmendel making sure no new ones slip in.

**Comment by jbrockmendel** [comment]
+1

File: setup.cfg [hunk 1]
```diff
@@ -31,5 +31,78 @@ select=
...
+    F707,
+    # F707: an except: block as not the last exception handler
+    F721,
+    # F721: doctest syntax error
+    F722,
+    # F722: syntax error in forward type annotation
+    F831,
+    # F831: duplicate argument name in function definition
+    F901
+    # F901: raise NotImplemented should be raise NotImplementedError
```
Comment (jbrockmendel) [inline setup.cfg]: we're currently checking for this as E901.  Are there two slightly different versions of this or is one of these a typo?

**Comment by bashtage** [comment]
E901 is A syntax error.


https://pycodestyle.readthedocs.io/en/latest/intro.html#error-codes

File: setup.cfg (hunk 1 above)
Comment (jbrockmendel) [reply setup.cfg]: Then I guess I must have made a typo way back when.  Might be worth updating the message attached to E901 up at the top

**Comment by jbrockmendel** [comment]
The Travis error here also showed up in #4936, not sure yet what it’s all about.

**Comment by jbrockmendel** [comment]
How did you find the list of currently-passing codes?  I’d like to check periodically if any new ones can be added to this list.

**Comment by jbrockmendel** [comment]
@josef-pkt this should be a nice dose of headache-prevention

**Comment by bashtage** [comment]
I just made a list of the violations and took the diff with the warnings
that pyflake or flake8 produces.

**Pull request merged**